src/nomad_simulations/schema_packages/basis_set.py

```diff
@@ -224,7 +224,47 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
         super().normalize(archive, logger)
         # self.name = self.m_def.name
 
+class AtomCenteredBasisSet(BasisSetComponent):
+    """
+    Defines an atom-centered basis set, using a single JSON quantity for all basis set types,
+    while allowing different AtomState references for each basis set type.
+    """
+
+    basis_set_data = Quantity(
+        type=JSON,  # Use JSON to store basis set information, including atom references
```

Comment:
@EBB2675 did you see my comment about this? it came up in my GH notifications, but I am not sure where it was stored. The comment was about using a repeating sub-section here instead of JSON

Reply:
@JFRudzinski I have just seen it (but only in the notification E-mail)

Alright, im on it :+1: